src/panel/direction/direction_panel.js

```diff
@@ -134,6 +134,9 @@ export default class DirectionPanel {
     this.panel.update()
     this.cleanDirection()
     UrlState.pushUrl()
+    document.querySelectorAll('.menu__button')[0].style.opacity = 1
+    document.querySelectorAll('.search_form')[0].style.opacity = 1
+    document.querySelectorAll('.direction_shortcut')[0].style.opacity = 1
```

Comment:
this looks a bit hackish :/
At least, we could put the elements to collapse in a single constant, and use `visibility` instead of `opacity` to prevent accessibility issues.

Reply:
+1 for visibility,
Is it ok if I toggle a class on the topmost container to hide/show these 3 items at once?

Reply:
and set `visibility` in css ? yes, I would prefer this solution.